chart/templates/role.yaml

```diff
@@ -118,6 +118,14 @@ rules:
     verbs: ["create", "delete", "patch", "update", "get", "list", "watch"]
   {{- end }}
   {{- end }}
+  {{- if .Values.integrations.istio.enabled }}
+  - apiGroups: [ "networking.istio.io" ]
+    resources: [ "destinationrules", "gateways", "virtualservices" ]
```

Comment:
Can we add `serviceentries` here for sleep mode even though we're not syncing them yet?

Reply:
added